src/menu_item.c

```diff
@@ -10079,56 +10133,56 @@ struct_8018D9E0_entry* find_8018D9E0_entry(s32 arg0) {
     return entry;
 }
 
-s32 func_800AAF70(s32 arg0) {
-    struct_8018D9E0_entry* temp;
-    temp = func_800AAEB4(arg0);
-    return temp->cursor;
+UNUSED s32 get_state_menu_item_character(s32 arg0) {
+    MenuItem* temp;
+    temp = get_menu_item_character(arg0);
+    return temp->state;
 }
 
-void func_800AAF94(struct_8018D9E0_entry* arg0, s32 arg1) {
-    struct_8018D9E0_entry* temp_v0;
+void move_menu_item_at_character_portrait(MenuItem* arg0, s32 character_id) {
```

Comment:
translate?

Is this like hover_cursor_over_character_portrait ?

Reply:
it's when you select a character it come at the middle at the screen it's that

Reply:
center_selected_character

Reply:
it's not necessary at the center it move the square of a character to another menu item